pallets/automation-time/src/lib.rs

```diff
@@ -325,31 +330,23 @@ pub mod pallet {
 			task_id: TaskId<T>,
 			error: DispatchError,
 		},
-		AutoCompoundDelegatorStakeSucceeded {
-			task_id: TaskId<T>,
-			amount: BalanceOf<T>,
-		},
 		AutoCompoundDelegatorStakeFailed {
+			who: AccountOf<T>,
 			task_id: TaskId<T>,
-			error_message: Vec<u8>,
-			error: DispatchErrorWithPostInfo,
-		},
-		AutoCompoundDelegatorStakeSkipped {
-			task_id: TaskId<T>,
-			message: Vec<u8>,
+			error: DispatchErrorWithDataMap,
 		},
```

Comment:
Since AutoCompoundDelegatorStakeSucceeded and AutoCompoundDelegatorStakeSkipped are gone, could this error event be replaced by TaskExecutionFailed as well? 

We don’t need to define `DispatchErrorWithDataMap` either, because delegation information will be contained in the `TaskTriggered` event.

Reply:
`TaskTriggered` is not defined yet.